.github/workflows/publish_image.yml

```
@@ -134,3 +134,59 @@ jobs:
     with:
       version: ${{ needs.settings.outputs.image_version }}
     secrets: inherit
+
+  update-remote-repo:
+    needs:
+      - settings
+      # - publish
+    runs-on: ubuntu-latest
+    env:
+      service_to_update: 'studio'
+      cli_repo_owner: supabase
+      cli_repo: cli
+      cli_repo_main_branch: 'develop'
+      cli_repo_pr_title: 'fix: Bump studio to the latest image version'
```

```suggestion
      cli_repo_pr_title: 'chore: Bump studio to the latest image version'
```

dependency upgrades should be chore not fix (or doesnt `chore` properly trigger a release)?